docs/features/hardware-offload/dpu-support.md

```diff
@@ -14,6 +14,118 @@ Similarly to Smart-NICs, a DPU follows the kernel switchdev model.
 In this model, every VF/PF net-device on the host has a corresponding representor net-device existing
 on the embedded CPU.
 
+
+## Diagrams of supported configurations Host + DPU
+
+### DHCP server serving on br-ex
+```
+┌───────────────────────────────────────────────────────────────────────────────────────────────────────────┐
+│Host                               ┌──────────────────────────────────────┐                                │
+│                                   │                                      │                                │
+│                                   │                                      │                                │
+│                                   │        OVNK IN DPU HOST MODE         │                                │
+│                                   │                                      │                                │
+│                                   │                                      │                                │
+│                                   │                                      │                                │
+│                                   └──────────────────────────────────────┘                                │
+│                                                                   ┌─────────────────────────────────────┐ │
+│                                                                   │ pf0hpf (ovnkube gateway interface)  │ │
+│                                                                   │            10.0.120.2/29            │ │
+│                                                                   └─────────────────────────────────────┘ │
+│                                                                                      ▲                    │
+│                                                                                      │                    │
+└──────────────────────────────────────────────────────────────────────────────────────┼────────────────────┘
+                                                                                       │                     
+┌──────────────────────────────────────────────────────────────────────────────────────┼────────────────────┐
+│DPU                                                                                   │                    │
+│                                                                                      ▼                    │
+│     ┌────────────────────┐           ┌──────────────────────────────────────────┬──────────┬────────┐     │
+│     │                    │           │                                          │pf0hpf_rep│        │     │
+│     │                    │           │ ┌─────────────────────────────────────┐  └──────────┘        │     │
+│     │                    │           │ │    DHCP Server Serving on br-ex     │                      │     │
```

Comment:
This DHCP server is not really in or on br-ex. It really exists as a host dnsmasq process that interacts vai the br-ex LOCAL port. Can we change the picture to depict that?

Reply:
done